pyprof/nvtx/nvmarker.py

```diff
@@ -312,15 +207,28 @@ def wrapper_func(*args, **kwargs):
             # Pop trace marker
             nvtx.range_pop()
 
+        if config.capture_input_ops:
+            dlprof.capture_outputs(dlprof.call_id, result)
+            # Store the callid -> op_name mapping
+            if config.func_stack_enabled:
```

Comment:
Don't need this IF (assuming you guard against the capture_input_ops=true func_stack_enabled=false case.

Reply:
Removed